servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/Completable.java

```diff
@@ -757,8 +798,45 @@ public final Completable beforeCancel(Runnable onCancel) {
      * </ul>
      * for Subscriptions/{@link Subscriber}s of the returned {@link Completable}. <strong>MUST NOT</strong> throw.
      * @return The new {@link Completable}.
+     *
+     * @see <a href="http://reactivex.io/documentation/operators/do.html">ReactiveX do operator.</a>
      */
     public final Completable beforeFinally(Runnable doFinally) {
+        return beforeFinally(TerminalSignalConsumer.from(doFinally));
+    }
+
+    /**
+     * Invokes the {@code whenFinally} {@link Runnable} argument <strong>before</strong> any of the following terminal
```

Comment:
Similar comment for other methods here and in other sources.

```suggestion
     * Invokes the corresponding method on {@code whenFinally} {@link TerminalSignalConsumer} argument <strong>before</strong> any of the following terminal
```